etc/ReplayOfflineConfiguration.py

```diff
@@ -225,12 +227,16 @@
 #############################
 addExpressConfig(tier0Config, "Express",
                  scenario=ppScenario,
+                 diskNode="T2_CH_CERN",
                  data_tiers=["FEVT"],
                  write_dqm=True,
                  alca_producers=["SiStripPCLHistos", "SiStripCalZeroBias", "SiStripCalMinBias", "SiStripCalMinBiasAAG",
                                  "TkAlMinBias", "LumiPixelsMinBias", "SiPixelCalZeroBias",
                                  "PromptCalibProd", "PromptCalibProdSiStrip", "PromptCalibProdSiPixelAli",
-                                 "PromptCalibProdSiStripGains", "PromptCalibProdSiStripGainsAAG", "PromptCalibProdSiPixel"
+                                 "PromptCalibProdSiStripGains", "PromptCalibProdSiStripGainsAAG",
+                                 "SiPixelCalSingleMuon", 
+                                 "PromptCalibProdSiPixel","PromptCalibProdSiPixelLA"
```

Comment:
```suggestion
                                 "PromptCalibProdSiPixel","PromptCalibProdSiPixelLorentzAngle"
```
That's how it is called:

https://github.com/cms-sw/cmssw/blob/master/Configuration/StandardSequences/python/AlCaRecoStreams_cff.py#L984

Reply:
Thanks, @mmusich ! Would you know why it is not also called 'PromptCalibProdSiPixelLorentzAngle' in https://github.com/cms-sw/cmssw/blob/master/Configuration/AlCa/python/autoPCL.py#L9  ?

Reply:
because I code it that way :D

Reply:
@mmusich would it be fine for you if we change that? In the case of the other PCL's one could just take their names from the autoPCL, and that's what I did here too, I think it would make our like a bit easier.

Reply:
I don't understand what's the advantange behind that change, that's just a dictionary with mute variables, but I don't have objections, provided you do it consistently.

Reply:
hmm apparently it is used by configBuilder after all: https://github.com/cms-sw/cmssw/blob/06f15e3ce91f53b005847b384d4027117598a9d7/Configuration/Applications/python/ConfigBuilder.py#L2071, so it seems that     
https://github.com/cms-sw/cmssw/pull/36182 (well, its backport) will be needed.